recipes/vscode-jupyter/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "vscode-jupyter" %}
+{% set version = "2020.12.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url: https://github.com/microsoft/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 99502db43f9f2ac4185d55ae51a70c81ffe9353983dcaca5cd1a7cf29f783a11
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - nodejs 12
+    - code-server >=3.9
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+  host:
+    - nodejs 12
+    - code-server >=3.9
+  run:
+    - code-server >=3.9
+    - python-jsonrpc-server 0.4.0
```

I'm confused that we need this as we had the extension running fine without conda without having this installed. Is this somehow bundled?
```suggestion
    - python-jsonrpc-server 0.4.0
```

---

Well, it's listed in the [`requirements.in`](https://github.com/microsoft/vscode-jupyter/blob/main/requirements.in), which I believe counts as runtime dependencies here.
I can't find any documentation on this anywhere in the repo, unfortunately. We can try to remove all Python runtime dependencies and test it locally. I can do that tomorrow.